githttp.go

```diff
@@ -38,6 +39,7 @@ func New(root string) *GitHttp {
 		GitBinPath:  "/usr/bin/git",
 		UploadPack:  true,
 		ReceivePack: true,
+		AutoCreate:  false,
```

Comment:
From a Go style perspective, given that `false` is already the zero value for `bool`, it might make sense to omit it instead of explicitly setting it to `false`.